dist/debian/debian/rules

```diff
@@ -3,7 +3,6 @@
 include /usr/share/dpkg/pkg-info.mk
 
 export PYBUILD_DISABLE=1
-jobs := $(shell echo $$DEB_BUILD_OPTIONS | sed -r "s/.*parallel=([0-9]+).*/-j\1/")
```

Comment:
Was added explicitly in bf27fdeaa2290c39dbc7160803cc30e82fb0ae10. So I don't think it's unused. /cc @syuu1228

Reply:
i am confused. you were referencing a change which only touched `dist/common/scripts/scylla_coredump_setup` and `scylla_post_install.sh`. neither of them is related to `debian/rules`. and the change was not added in the commit you were referencing.

Reply:
I had the wrong commit in the paste buffer.

The correct commit is 782ebcece4f860238abc4dbc5dc4f81fdd1f02f9.

Reply:
i referenced exact the same commit in the commit message. and explained why was is not used anymore.